spec/factories/projects.rb

```diff
@@ -7,6 +7,21 @@
     initialize_with { Tmuxinator::Project.new(file) }
   end
 
+  factory :project_with_force_attach, :class => Tmuxinator::Project do
+    transient do
+      file { YAML.load(File.read("#{File.expand_path("spec/fixtures/detach.yml")}")) }
+    end
+
+    initialize_with { Tmuxinator::Project.new(file, {:force_attach => true}) }
+  end
+
+  factory :project_with_force_detach, :class => Tmuxinator::Project do
+    transient do
+      file { YAML.load(File.read("#{File.expand_path("spec/fixtures/detach.yml")}")) }
```

Comment:
Line is too long. [86/80]<br>Prefer single-quoted strings inside interpolations.